src/main/java/com/sparta/popupstore/domain/promotionalevent/dto/response/PromotionEventResponseDto.java

```diff
@@ -0,0 +1,40 @@
+package com.sparta.popupstore.domain.promotionalevent.dto.response;
+
+import com.sparta.popupstore.domain.popupstore.entity.PopupStore;
+import com.sparta.popupstore.domain.promotionalevent.entity.PromotionalEvent;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Getter;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
+
+@Getter
+public class PromotionEventResponseDto {
```

Comment:
```suggestion
public class PromotionEventFindResponseDto {
```
dto 이름에 CRUD 메서드도 넣어주세요.